### DISCUSSION TITLE
Exportation pdf sur ipad

### DISCUSSION THREAD

**Discussion by kellyrls** [saber-notes/saber#1104] [Q&A]
Bonjour, 
J’ai un iPad et je viens d’installer l’application Saber. Je n’arrive pas à exporter en pdf les fichiers. Je suis pourtant bien connectée à un Nextcloud. Comment faire pour résoudre le problème. La flèche tourne s’arrête et retourne. 
merci de votre aide.

**Comment by Rajstopy** [comment]
I’ve just installed this app and noticed the exact same problem. I was looking for such an app able to rely on Nextcloud but without PDF export it doesn’t make sense anymore. Hope this will be fixed.
